src/cli/cli_main.py

```diff
@@ -103,17 +109,20 @@ def ls_data() -> None:
     """
     # fetch list of registered datasets
     import palimpzest as pz
+    # Retrieve all datasets currently registered in the system
+    
     ds = pz.DataDirectory().listRegisteredDatasets()
 
     # construct table for printing
     table = [["Name", "Type", "Path"]]
     for path, descriptor in ds:
         table.append([path, descriptor[0], descriptor[1]])
 
-    # print table of registered datasets
+    # Use PrettyTable to format and display the dataset information in a table format
+    
     t = PrettyTable(table[0])
     t.add_rows(table[1:])
-    _print_msg(t)
+    _print_msg(t)      # Print additional information, like the total number of datasets
```

Comment:
The position of this comment is misleading - this statement does not print the total number of datasets, but the following one does.